src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -37,6 +37,13 @@
             <div class="sf-toolbar-info-piece">
                 <b>Actions</b>
                 <span><a href="{{ collector.logoutUrl }}">Logout</a></span>
+                {% if 'ROLE_PREVIOUS_ADMIN' in collector.roles %}
+                    - <span>
+                        <a href="{{ path(profile.getcollector('request').requestattributes.get('_route'), {'_switch_user': '_exit'}) }}">
```

Comment:
Beware that the `_switch_user` query param is configurable for security reasons. See http://symfony.com/doc/current/cookbook/security/impersonating_user.html

![impersonate](https://cloud.githubusercontent.com/assets/73419/11850395/593ddc60-a42d-11e5-8703-e557a03f4c97.png)